### ISSUE TITLE
Feature Request: GlassPlayer GlassCommander integration

### ISSUE THREAD

**Issue by alexolivan** [ElvishArtisan/GlassCoder#13]
Hi!

Not sure wether this is the rigth place to ask for it.... hope I'm not spamming:

I've been very very happy learning how to use GlassCommander as a a GUI tool to manage Output streamings... very very nice point that makes glasscommander match the best encoding tools I've seen.

However, considering the existence of GlassPlayer GUI, I immediatelly though that integrating GlassPlayer instances into GlassCommander,  it would result into the best swiss-army knife streaming management-tool ever.

Cheers!

**Comment by ElvishArtisan** [comment]
You've come to the right place. :smile:

However, I'm a bit unsure what you mean by 'integration'. They _are_ integrated already, inasmuch as instances created by GlassGui are usable in GlassCommander. Ca you give us an example of what you have in mind?

**Comment by alexolivan** [comment]
Hi!

The thing is that I'm not refering to the coding software/project (which I found perfectly integrated as you say), GlassCoder, but instead the 'companion'/'parallel' PLAYBACK code/project: GlassPlayer (I posted this same issue on GlassPlyer Repo... thats why I was affraid of not Spamming or not using the right repo to comment)

I mean... It would be nice If glassplayer had an "--instance-name" modifier the way glasscoder has, so it, a playback/RX instance could be openend alongside the encoding instances (I know, I know... this is far easier said than done :-) AFIK GlassPlayer and GlassCoder are distinct pieces of code)
A tool that manages in/out streamings in a centralized/organized manner, is as powerful/preofesional, as rare, outside commercial solutions... GlassCommander is an incredible piece of code doing half (TX) of that use-case!

GlassCommander looks to me like some cool tools I've seen on some Win-Based Professional Radio Automation Software (I have SAM Broadcaster in mind) ... sure Rivendell would deserve to have some kind of GlassCommander as an integral part of it... a  kind of 'RDStreamSlots' or 'RDStreamManager' :-P

Thank you!
Best regards.
